pages/work/enigma.vue

```diff
@@ -0,0 +1,147 @@
+          state when Ship Shape was brought in to act as the development team
+          that would create a production-ready product. Enigma already had
+          customers signed up and ready to start utilizing the application for
+          their recruiting needs, but still had a substantial roadmap of
+          features in need of implementation, including the full assessment
+          workflow and administration-related tasks. Ship Shape’s challenge was
+          to implement the crucial features necessary to launch Enigma and meet
+          customers’ various requests.
+        </p>
+
+        <h2>Solution</h2>
+
+        <h3>Next.js</h3>
+
+        <p>
+          The development team made use of
+          <a href="/https://nextjs.org/">Next.js</a> to prioritize quick loading
+          times on the client side of this application. The server-side
+          rendering capabilities of Next.js allows the Enigma application to
+          retrieve web pages and any corresponding data in one go, eliminating
+          the need for multiple network requests and subsequent delays in data
+          population.
+        </p>
+
+        <h3>Shepherd Tour</h3>
+
+        <p>
+          The Ship Shape team also had the opportunity to integrate one of our
+          own open-source projects,
+          <a href="https://shepherdjs.dev/docs/">Shepherd</a>, into the Enigma
+          platform. The Shepherd library offers a way to guide Enigma users
+          through a tour of the application, highlighting the key features users
+          should familiarize themselves with in order to use the app at its full
+          potential.
+        </p>
+
+        <h3>Cypress Testing</h3>
+
+        <p>
+          The development team also incorporated
+          <a href="https://www.cypress.io/">Cypress</a>, an integration testing
+          framework, into the Enigma application. This end-to-end testing
+          technology allowed the development team to deliver features to users
+          with confidence that critical tasks aren’t disrupted by bugs.
+        </p>
+
+        <h3>Key Application Features</h3>
+
+        <p>
+          In order to successfully launch this application, the Enigma and Ship
+          Shape development teams focused on implementing the key features that
+          are central to making Enigma the unique web application that it is.
+          These features included the following crucial workflows:
+        </p>
+
+        <ul class="ml-16">
+          <li class="list-disc">
+            Authentication processes that allow candidates and employers to
+            access their respective accounts
+          </li>
+          <li class="list-disc">
+            A candidate’s ability to work through various assessments assigned
+            by potential employers
+          </li>
+          <li class="list-disc">
+            An employer’s ability to create, view, assign, archive, and delete
```

Comment:
Is creating different types of questions a unique feature too (MC/blank line/etc)? or too specific for this list maybe?

Reply:
Hmm, I get where you're coming from, although I don't know if that fits here since the employer doesn't have the ability to create different types of questions. May leave off.